samcli/local/docker/lambda_container.py

```diff
@@ -145,6 +147,13 @@ def _get_entry_point(runtime, debug_port=None, debug_args=None):
                         "/var/runtime/lib/LambdaJavaRTEntry-1.0.jar",
                    ]
 
+        elif runtime == Runtime.go1x.value:
+            entrypoint = ["/var/runtime/aws-lambda-go"] \
+                + [
+                    "-debug=true",
+                    "-delvePort=" + debug_port
```

Comment:
Can we set `delvePath` here? This will allow SAM CLI to continue to work even if Docker image changed their default value of `delvePath`.

I am okay with a hard-coded value like `/tmp/samcli_debug_files` or something